PluginImportedCheck.sh

```diff
@@ -0,0 +1,10 @@
+plugin_dir=$1
+for dir in `ls "./apm-sniffer/$plugin_dir/"`; do
+	for f in `find ./apm-sniffer/$plugin_dir/$dir -name *Instrumentation.java `; do
+		NUM=`head -400 $f | grep import |grep -v net.bytebuddy. | grep -v org.apache.skywalking. |grep -v java.| wc -l`;
```

Comment:
`grep import` => `grep ^import` should be more accurately

Reply:
Yes